taxi/views.py

```diff
@@ -79,6 +86,18 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
     success_url = reverse_lazy("taxi:car-list")
 
 
+class CarDriverManageView(LoginRequiredMixin, generic.View):
+    @staticmethod
+    def post(request: HttpRequest, pk: int):
+        car = get_object_or_404(Car, pk=pk)
+
+        if car.drivers.filter(id=request.user.id).exists():
+            car.drivers.remove(request.user)
+        else:
+            car.drivers.add(request.user)
+        return render(request, "taxi/car_detail.html", {"car": car})
```

Comment:
You don't need `staticmethod` in CBV(Class-based views) in django.

[Link](https://docs.djangoproject.com/en/5.1/topics/class-based-views/intro/)